internal/database/dialer/cockroachdb/client_bridge.go

```diff
@@ -59,6 +59,27 @@ func (c *client) FindBridgeTransactions(ctx context.Context, query schema.Bridge
 
 	databaseClient := c.database.WithContext(ctx)
 
+	const limit = 100
+
+	if query.Cursor != nil {
+		var cursor table.BridgeTransaction
+		if err := databaseClient.Where(`"id" = ?`, query.Cursor.String()).First(&cursor).Error; err != nil {
+			return nil, fmt.Errorf("query cursor: %w", err)
+		}
+
+		// TODO Need a better cursor implementation.
```

Comment:
Because bridge transactions originate from two chains, the order of the block number and transaction index cannot be used together.